apiml-security-common/src/test/java/org/zowe/apiml/security/common/login/ChangePasswordRequestTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package org.zowe.apiml.security.common.login;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ChangePasswordRequestTest {
```

Comment:
Do we need this test?

Reply:
LoginRequestTest is also tested, I've done it to avoid drop of code coverage